src/router/index.ts

```diff
@@ -1,9 +1,9 @@
 import { createRouter, createWebHistory } from 'vue-router'
-import HomeView from '@/views/HomeView.vue'
-import IdeasView from '@/views/IdeasView.vue'
-import TeamView from '@/views/TeamView.vue'
-import ELSIView from '@/views/ELSIView.vue'
-import GroupAView from '@/views/enzymosome/GroupAView.vue'
+const HomeView = () => import('@/views/HomeView.vue')
+const IdeasView = () => import('@/views/IdeasView.vue')
+const TeamView = () => import('@/views/TeamView.vue')
+const ELSIView = () => import('@/views/ELSIView.vue')
+const GroupAView = () => import('@/views/enzymosome/GroupAView.vue')
```

Comment:
Yeah, I was gonna ask you about this. But it seems like you figured out the lazy loading so lesgoooo! 💯